src/schema/objects/enums.yaml

```diff
@@ -0,0 +1,1113 @@
+    - $ref: objects.enums.UNCInfant1V21.value
+    - $ref: objects.enums.UNCInfant2V21.value
+    - $ref: objects.enums.UNCInfant0V22.value
+    - $ref: objects.enums.UNCInfant1V22.value
+    - $ref: objects.enums.UNCInfant2V22.value
+    - $ref: objects.enums.UNCInfant0V23.value
+    - $ref: objects.enums.UNCInfant1V23.value
+    - $ref: objects.enums.UNCInfant2V23.value
+_iEEGCoordSys:
+  type: string
+  enum:
+    - $ref: objects.enums.Pixels.value
+    - $ref: objects.enums.ACPC.value
+    - $ref: objects.enums.ScanRAS.value
+    - $ref: objects.enums.Other.value
+left_hemisphere:
+  value: 'L'
+  display_name: Left Hemisphere
+  description: |
+    A left hemibrain image.
+right_hemisphere:
+  value: 'R'
+  display_name: Right Hemisphere
+  description: |
+    A right hemibrain image.
+CASL:
+  value: CASL
+  display_name: Continuous arterial spin labeling
+  description: |
+    Continuous arterial spin labeling was employed.
+PCASL:
+  value: PCASL
+  display_name: Pseudo-continuous arterial spin labeling
+  description: |
+    Pseudo-continuous arterial spin labeling was employed.
+PASL:
+  value: PASL
+  display_name: Pulsed arterial spin labeling
+  description: |
+    Pulsed arterial spin labeling was employed.
+Separate:
+  value: Separate
+  display_name: Separate
+  description: |
+    A separate `m0scan` file is present.
+Included:
+  value: Included
+  display_name: Included
+  description: |
+    An m0scan volume is contained within the associated `asl` file.
+Estimate:
+  value: Estimate
+  display_name: Estimate
+  description: |
+    A single whole-brain M0 value is provided in the metadata.
+Absent:
+  value: Absent
+  display_name: Absent
+  description: |
+    No specific M0 information is present.
+TwoD:
+  value: 2D
+  display_name: Two-dimensional
+  description: |
+    Two-dimensional MR acquisition.
+ThreeD:
+  value: 3D
+  display_name: Three-dimensional
+  description: |
+    Three-dimensional MR acquisition.
+HARD:
+  value: HARD
+  display_name: Hard pulse
+  description: |
+    A very brief, strong, rectangular pulse.
+GAUSSIAN:
+  value: GAUSSIAN
+  display_name: Gaussian pulse
+  description: |
+    A Gaussian pulse.
+GAUSSHANN:
+  value: GAUSSHANN
+  display_name: Gaussian-Hanning pulse.
+  description: |
+    A Gaussian pulse with a Hanning window.
+SINC:
+  value: SINC
+  display_name: Sinc pulse
+  description: |
+    A sinc-shaped pulse.
+SINCHANN:
+  value: SINCHANN
+  display_name: Sinc-Hanning pulse
+  description: |
+    A sinc-shaped pulse with a Hanning window.
+SINCGAUSS:
+  value: SINCGAUSS
+  display_name: Sinc-Gauss pulse
+  description: |
+    A sinc-shaped pulse with a Gaussian window.
+FERMI:
+  value: FERMI
+  display_name: Fermi pulse
+  description: |
+    A Fermi-shaped pulse.
+i:
+  value: i
+  display_name: i
+  description: |
+    Phase/Slice encoding direction is along the first axis of the data in the NIFTI file,
+    with phase encoding going from the zero index to the maximum index.
```

Comment:
```suggestion
    The encoding direction is along the first axis of the data in the NIFTI file,
    and the encoding value increases from the zero index to the maximum index.
```

Reply:
I'm happy to just remove mention of phase and slice, as in your suggestion.